src/setup/globals.js

```diff
@@ -51,7 +51,7 @@ window.ProcessMaker = {
   },
   modeler: {
     process: {
-      id: 1,
+      id: 4,
```

Comment:
why this was changed to 4?

Reply:
It was for testing purposes, It has been restored